mcpi/minecraft.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, connection, packagePrefix):
     def getPos(self, id):
         """Get entity position (entityId:int) => Vec3"""
         s = self.conn.sendReceive(self.pkg + ".getPos", id)
-        return Vec3(*map(float, s.split(",")))
+        return Vec3(*list(map(float, s.split(","))))
```

Comment:
Are you sure this is required? It seems that list unpacking works with iterators as well.

```python
>>> def make(a, b, c):
...     return (a, b, c)
...
>>> make(*map(float, '1,2,3'.split(',')))
(1.0, 2.0, 3.0)
```